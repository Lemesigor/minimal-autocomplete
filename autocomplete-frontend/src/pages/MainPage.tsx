import {Header} from "../components/Header/Header";
import {Form} from "../components/Form/Form";

export const MainPage = () => {
    return (
        <main>
            <Header/>
            <Form/>
        </main>
    )
}