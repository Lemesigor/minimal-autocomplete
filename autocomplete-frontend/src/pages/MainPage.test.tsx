import {render} from "@testing-library/react";

import {MainPage} from "./MainPage";

describe("MainPage", () => {
    it("should render the main page", () => {
        const {container} = render(<MainPage />);
        expect(container).toMatchSnapshot();
    });
})
