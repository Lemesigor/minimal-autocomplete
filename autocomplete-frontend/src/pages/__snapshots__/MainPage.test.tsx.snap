// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage should render the main page 1`] = `
<div>
  <main>
    <header
      class="css-1188xm2"
    >
      <h2
        class="css-rnq8sf"
      >
        Autocomplete
      </h2>
    </header>
    <section
      class="css-1a6pxyc"
    >
      <input
        placeholder="Search"
        type="text"
      />
    </section>
  </main>
</div>
`;
